example/client/main.go

```diff
@@ -1,75 +1,68 @@
 package main
 
 import (
-	"bytes"
 	"crypto/tls"
-	"crypto/x509"
-	"flag"
-	"io"
-	"net/http"
-	"sync"
-
-	"github.com/lucas-clemente/quic-go/http3"
-	"github.com/lucas-clemente/quic-go/internal/testdata"
-	"github.com/lucas-clemente/quic-go/internal/utils"
+	"fmt"
```

Comment:
Probably revert the changes to this file?

Reply:
🤦🏻‍♂